s3/test-results/main.tf

```
@@ -0,0 +1,26 @@
+# ### test-results-s3bucket
+# S3 Bucket name will have
+# "region-environment_name" prepended
+
+locals {
+  bucket_name = "${var.tiny_environment_identifier}-test_results"
```

```suggestion
  bucket_name = "${var.tiny_environment_identifier}-test-results"
```
Could we use a dash instead of an underscore here, to be consistent with the other buckets? All good otherwise.